server/tests/audit_math/test_bravo.py

```diff
@@ -379,6 +379,44 @@ def test_compute_risk_empty(contests):
         )
 
 
+def test_tied_contest():
+    contest_data = {
+        "cand1": 500,
+        "cand2": 500,
+        "ballots": 1000,
+        "numWinners": 1,
+        "votesAllowed": 1,
+    }
+
+    contest = Contest("Tied Contest", contest_data)
+
+    sample_results = {}
+
+    sample_options = bravo.get_sample_size(RISK_LIMIT, contest, sample_results)
+
+    assert 0.7 not in sample_options
+    assert 0.8 not in sample_options
+    assert 0.9 not in sample_options
+    assert sample_options["asn"]["size"] == contest.ballots
+    assert sample_options["asn"]["prob"] == 1.0
+
+    computed_p, res = bravo.compute_risk(RISK_LIMIT, contest, sample_results)
+
+    assert computed_p[("cand1", "cand2")] > RISK_LIMIT
+    assert not res
+
+    # Now do a full hand recount
+    sample_results = {
+        "cand1": 501,
+        "cand2": 499,
+    }
+
+    computed_p, res = bravo.compute_risk(RISK_LIMIT, contest, sample_results)
+
+    assert not computed_p[("cand1", "cand2")]
```

Comment:
similar nit as to above - are you asserting that it's `None` or `0`?

Reply:
0. I will change it to a `==`